test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/PackageSignatureVerifierTests.cs

```diff
@@ -0,0 +1,782 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if IS_SIGNING_SUPPORTED
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography.Pkcs;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NuGet.Common;
+using NuGet.Packaging.Signing;
+using NuGet.Test.Utility;
+using Org.BouncyCastle.Asn1;
+using Org.BouncyCastle.Asn1.X509;
+using Org.BouncyCastle.Crypto;
+using Test.Utility;
+using Test.Utility.Signing;
+using Xunit;
+using BcAccuracy = Org.BouncyCastle.Asn1.Tsp.Accuracy;
+using BcX509Certificate = Org.BouncyCastle.X509.X509Certificate;
+using HashAlgorithmName = NuGet.Common.HashAlgorithmName;
+
+namespace NuGet.Packaging.FuncTest
+{
+    [Collection(SigningTestCollection.Name)]
+    public class PackageSignatureVerifierTests
+    {
+        private readonly SigningTestFixture _testFixture;
+
+        public PackageSignatureVerifierTests(SigningTestFixture fixture)
+        {
+            _testFixture = fixture ?? throw new ArgumentNullException(nameof(fixture));
+        }
+
+        public void Constructor_WhenArgumentIsNull_Throws()
```

Comment:
Does it need `[CIOnlyFact]` attribute? Not sure if it's triggered.

Reply:
Good catch